src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -446,6 +440,12 @@ class QueryCompiler(cq : ConjunctiveQuery, ss: CompilationState) {
         ${ optionalClause("WHERE", generateWhereClause(body, "")) }${groupbyStr}${limitStr}"""
   }
 
+  def generateSQL(useAlias: Boolean) = {
+    val head = generateSQLHead(useAlias)
+    val body = cq.bodies map generateSQLBody mkString("\nUNION ALL\n")
```

Comment:
Does joining only the bodies with `UNION ALL` correct?  I think each part should start with `SELECT`?  Do we have test cases with more than one body?